pyfr/solvers/euler/elements.py

```diff
@@ -98,12 +98,15 @@ def set_backend(self, *args, **kwargs):
             # Template arguments
             fpts_in_upts = self.basis.fpts_in_upts
             self.nefpts = self.nupts if fpts_in_upts else self.nupts + self.nfpts
+            ub = self.basis.ubasis
+            meanwts = ub.invvdm[:, 0] / np.sum(ub.invvdm[:, 0])
```

Comment:
What syntax are you thinking? `meanwts` here is the vector of quadrature weights that compute the mean. This could also be replaced with just the quadrature weights (normalized to 1) instead of using invvdm, that might be more obvious.